posts/2022-12-14-android-ndk-update.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Updating the Android NDK in Rust <version>"
+author: Android Platform Team
+description: "Modernizing Android support in Rust"
+---
+
+We are pleased to announce that Android platform support in Rust will be
+modernized in Rust <version> as we update the target NDK from r17 to r25.  As a
+consequence the minimum supported API level will increase from 15 (Jelly Bean)
+to 19 (KitKat).
+
+In NDK r23 Android switched to using LLVMs `libunwind` for all architectures.
+This meant that
+1. If a project were to target NDK r23 or newer with previous versions of Rust
+   a workaround would be required to redirect attempts to link against `libgcc`
```

Comment:
It might be useful to link to some documentation explaining the workaround in more detail, e.g. https://github.com/rust-lang/rust/pull/85806#issuecomment-1096266946. This does double-duty of 1. making sure it's clear what workaround the blog post is referring to and 2. making it easier for anyone who reads this and _wants_ to apply the workaround to find it.

Reply:
Done